query.css

```diff
@@ -0,0 +1,217 @@
+/* Large mobile screen */
+@media only screen and (max-width: 992px) {
+  .menu-links {
+    display: none;
+  }
+
+  .work-projects {
+    height: 600px;
+  }
+
+  .header-img-a {
+    height: 250px;
+    border-radius: 5px;
+  }
+
+  .header-img-b {
+    height: 250px;
+  }
+
+  .contact-container {
+    background-size: auto;
+  }
+}
+
+/* Desktop query portfolio version */
+@media only screen and (min-width: 992px) {
+  .off-menu {
+    display: none;
+  }
+
+  .menu-contianer {
+    height: 100%;
+    padding: 10px 12px;
+    flex-grow: 1;
+    display: flex;
+    flex-direction: row;
+  }
+
+  .menu-item {
+    display: flex;
+    width: 100%;
+    margin-right: 100px;
+    align-items: center;
+    justify-content: flex-end;
+    flex-direction: row;
+    gap: 10px;
+  }
+
+  .menu-links {
+    list-style: none;
+    margin: 0 10px;
+    transition: color 250ms ease;
+  }
+
+  .menu-links:hover {
+    color: #6070ff;
+  }
+
+  .headline-container {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    padding: 320px 300px;
+  }
+
+  .headline-title {
+    width: 470px;
+    height: 180px;
+    display: flex;
+    justify-content: start;
+    margin: 5px;
+  }
+
+  .title {
+    font-style: normal;
+    font-size: 40px;
+    line-height: 52px;
+    padding: 8px;
+  }
+
+  .description-item {
+    font-size: 20px;
+    font-style: normal;
+    line-height: 30px;
+    letter-spacing: 3px;
+    text-align: justify;
+  }
+
+  .link-item {
+    font-size: 16px;
+    font-style: normal;
+    line-height: 24px;
+  }
+
+  .link {
+    color: #7f8cff;
+  }
+
+  .work-projects {
+    border-radius: 25px;
+    display: flex;
+    margin: 50px 150px 100px 150px;
+    padding: 10px;
+    transition: box-shadow 450ms ease-in-out;
+    transition: transform 350ms ease-in;
+  }
+
+  .work-projects-b {
+    border-radius: 25px;
+    display: flex;
+    margin: 50px 150px 100px 150px;
+    padding: 10px;
+    height: 550px;
+    flex-direction: row-reverse;
+    transition: box-shadow 450ms ease-in-out, transform 350ms ease-in;
+  }
+
+  .card-content {
+    flex-grow: 1;
+    height: 100%;
+  }
+
+  .project-description {
+    display: flex;
+    flex-direction: column;
+    padding: 10px;
+    gap: 10px;
+    height: 100%;
+  }
+
+  .description-title {
+    padding: 12px;
+    font-size: 36px;
+  }
+
+  .description-item-b {
+    font-size: 18px;
+    line-height: 24px;
+    font-weight: 400;
+    color: #344563;
+    letter-spacing: 0;
+  }
+
+  .project-article-container {
+    margin-top: 40px;
+  }
+
+  .about-project {
+    display: flex;
+  }
+
+  .about-items-a {
+    flex-grow: 1;
+    width: 50%;
```

Comment:
- [ ] [REQUIRED] Please ensure that this section, the left section of the (About Myself section) takes a width of 40% so as to match the original design. Yours is 50%